features/graph/api/src/main/java/org/opennms/netmgt/graph/api/search/SearchSuggestion.java

```diff
@@ -80,4 +85,38 @@ public String getProvider() {
     public void setProvider(String provider) {
         this.provider = provider;
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        SearchSuggestion that = (SearchSuggestion) o;
+        return com.google.common.base.Objects.equal(context, that.context) &&
```

Comment:
There is no need to use google guava here, java 8 has Objects.equals() as well

Reply:
fixed.